frontend/simple/components/GroupMembers.vue

```diff
@@ -0,0 +1,58 @@
+<script>
+import {mapGetters} from 'vuex'
+export default {
+  name: 'GroupMembers',
+  props: {
+    group: Object
+  },
+  methods: {
+    invite (groupId) {
+      this.$store.commit('setCurrentGroupId', groupId)
+      this.$router.push({path: '/invite'})
+    }
+  },
+  computed: {
+    members () {
+      var members = this.$store.getters.membersForGroup()
+      const usernames = Object.keys(members)
+      usernames.map(function (username, idx) {
+        members[username] = {
+          attrs: members[username]
+        }
+      })
+      return members
+    },
+    ...mapGetters(['currentGroup'])
+  }
+}
+</script>
+<template>
+  <section>
+    <h3 class="title is-3"><i18n>Members</i18n></h3>
+    <ul>
+      <li v-for="(member, username) in members">
+        {{ username }}
+      </li>
+    </ul>
+    <a class="button invite-button" @click.prevent="invite(group.id)">
+      <span class="icon"><i class='fa fa-plus'></i></span>
+    </a>
+  </section>
+</template>
```

Comment:
Please follow the convention of all the other .vue files so that the `<template>` section is at the top of the file. Note this needs to be done for `GroupSettings.vue` and maybe others as well.